controllers/installation_controller.go

```diff
@@ -312,11 +312,16 @@ func (r *InstallationReconciler) ReconcileHelmCharts(ctx context.Context, in *v1
 		return nil
 	}
 
+  if in.Status.State == v1beta1.InstallationStateFailed {
```

Comment:
There is something wrong with the indentation here[

Reply:
sorry, I could swear I ran gofmt...